easybuild/easyblocks/h/hdf5.py

```diff
@@ -92,13 +92,14 @@ def sanity_check_step(self):
         else:
             extra_binaries = ["bin/%s" % x for x in ["h5cc", "h5fc"]]
 
+        SHLIB_EXT = get_shared_lib_ext()
         custom_paths = {
                         'files': ["bin/h5%s" % x for x in ["2gif", "c++", "copy", "debug", "diff",
                                                            "dump", "import", "jam","ls", "mkgrp",
                                                            "perf_serial", "redeploy", "repack",
                                                            "repart", "stat", "unjam"]] +
                                  ["bin/gif2h5"] + extra_binaries +
-                                 ["lib/libhdf5%s.so" % x for x in ["_cpp", "_fortran", "_hl_cpp",
+                                 ["lib/libhdf5%s.%s" % (x,SHLIB_EXT) for x in ["_cpp", "_fortran", "_hl_cpp",
```

Comment:
nitpicking:

* space after `,`
* align the line below accordingly (see similar line with list split across multiple lines above)